src/logging.cpp

```diff
@@ -182,6 +182,9 @@ const CLogCategoryDesc LogCategories[] =
     {BCLog::BLOCKSTORE, "blockstorage"},
     {BCLog::TXRECONCILIATION, "txreconciliation"},
     {BCLog::SCAN, "scan"},
+    // Add new entries before this line.
+
+    {BCLog::TEST, "test"}, // logging testing only
```

Comment:
What about calling it `LOGTEST` instead of `TEST`? Makes the purpose more clear and also keeps the `TEST` name free if we want to use it later?